src/stores.ts

```diff
@@ -49,11 +49,13 @@ export function newFeatureId(gj: Scheme): number {
 }
 
 export function deleteIntervention(id: number) {
+  console.log(`Deleting intervention ${id}`);
   gjScheme.update((gj) => {
     gj.features = gj.features.filter((f) => f.id != id);
     return gj;
   });
   formOpen.set(null);
   mapHover.set(null);
   sidebarHover.set(null);
+  openFromSidebar.set(null);
```

Comment:
Why is this necessary? https://github.com/acteng/atip/blob/b1b384908390f21b976777fed5e882c8caa80c76/src/lib/draw/AttributeMode.svelte#L37 is the place `openFromSidebar` gets used. The intention here is to only rerun this reactive block when that store changes. But Svelte doesn't work by specifying the variables to watch; when anything in the reactive statement changes, the whole thing reruns. So what's happening here is that `gjScheme` gets updated, deleting the feature. `openFromSidebar` is still set to something, and so looking up the feature by ID fails. We fix by remembering to clear this other store too -- aka, clean up all references to the deleted object.